install_files/appstream/io.github.openrazer.openrazer.metainfo.xml

```diff
@@ -74,6 +74,7 @@
     <modalias>usb:v1532p0225d*</modalias>
     <modalias>usb:v1532p0226d*</modalias>
     <modalias>usb:v1532p0228d*</modalias>
+    <modalias>usb:v1532p0228e*</modalias>
```

Comment:
This is wrong but you can just run `./scripts/generate_appstream_file.sh` to update this file